governator-test-junit/src/main/java/com/netflix/governator/guice/test/TestInjectorCreationMode.java

```diff
@@ -0,0 +1,6 @@
+package com.netflix.governator.guice.test;
+
+public enum TestInjectorCreationMode {
+    BEFORE_CLASS,
```

Comment:
I matched the Spring equivalent enum. Plus I did like that BEFORE_EACH_TEST_METHOD was very explicit about when the Injector was being created. BEFORE is a little ambiguous. Also worth noting that this is intended to be completely separate of JUnit as we will want to support TestNG, Spock, and Junit5 as well. Only the JUnit4Runner is tied to JUnit classes and conventions.